drift/app.py

```diff
@@ -1,13 +1,51 @@
-import connexion
+import os
 import logging
 
+import connexion
+import watchtower
+from boto3.session import Session
+
 from drift import config
 from drift.views import v1
 from drift.error import handle_http_error
 from drift.exceptions import HTTPError
 from drift.metrics_registry import create_prometheus_registry_dir
 
 
+def _setup_cw_logging(logger):  # pragma: no cover
+    """
+    initialize cloudwatch logging
+
+    from https://github.com/RedHatInsights/cloudwatch-test
+    """
+    key_id = os.environ.get("CW_AWS_ACCESS_KEY_ID")
+    secret = os.environ.get("CW_AWS_SECRET_ACCESS_KEY")
+    if not (key_id and secret):
+        logger.info("CloudWatch logging disabled due to missing access key")
```

Comment:
Should we error "in production" if these are not set? We could miss that we are not capturing logs in production which would hurt our ability to debug.

Reply:
I'm a little hesitant to add that only because I don't want to add "this only runs on non-dev deployments" code.

We could add an internal monitoring job to confirm logs are landing maybe?

Reply:
@kylape What have been y'alls approach to this question I have around ensuring logs are landing in production?